packages/react-server-dom-webpack/node-register.js

```diff
@@ -7,4 +7,4 @@
  * @flow
  */
 
-export * from './src/ReactFlightWebpackNodeRegister';
+module.exports = require('./src/ReactFlightWebpackNodeRegister');
```

Comment:
This is just because jest's treatment of ESM is not the same as rollup. So I switched this to use the jest way, and then just skip this file in the rollup bundling.